src/lib/core/exception.cc

```diff
@@ -430,6 +443,18 @@ BuildRaftError(const char *file, unsigned line, const char *format, ...)
 	return e;
 }
 
+struct error *
+BuildHandleIsClosed(const char *file, unsigned line, const char *format, ...)
+{
+	void *ptr = xmalloc(sizeof(HandleIsClosed));
```

Comment:
Nit: other errors use `BuildAlloc`. Let's use `BuildAlloc` here for consistency.

Reply:
No problem, changed.